### ISSUE TITLE
ip-1.1.0 benchmarks not compiling

### ISSUE THREAD

**Issue by juhp** [andrewthad/haskell-ip#22]
In the Stackage Nightly build, for 1.1.0 the benchmarks fail to compile

```
test/IPv4TextVariableBuilder.hs:10:1: error:
    Could not find module ‘Data.Text.Builder.Variable’
    it is a hidden module in the package ‘ip-1.1.0’
    Use -v to see a list of the files searched for.
   |
10 | import qualified Data.Text.Builder.Variable as VB
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by juhp** [comment]
See also athanclark/attoparsec-uri/issues/1

**Comment by andrewthad** [comment]
I'll get this fixed. Thanks.

**Comment by andrewthad** [comment]
This might be fixed by 6b4564e0a928e17c6de8123f07177bed77b1645a. I need to confirm it on another machine.
